apps/communications/contacts/elements/settings.html

```diff
@@ -34,28 +34,6 @@ <h1 data-l10n-id="settings"></h1>
               </button>
             </li>
           </ul>
-          <gaia-subheader id="fb-header" skin="organic">
```

Comment:
We are keeping all rules of FB in the same file, and extracting the code we need for the rest of functionality. The idea is to remove the whole FB code at the end, but by now we are focus just in removing dependencies.